tests/integration/ceph_base_object_test.go

```diff
@@ -212,7 +213,10 @@ func testObjectStoreOperations(s suite.Suite, helper *clients.TestClient, k8sh *
 			assert.NotEmpty(s.T(), objectStore.Status.Info["endpoint"])
 			break
 		}
-		assert.NotEqual(t, 10, i)
+		if i == 10 {
+			t.Fatal("bucket status check failed. status is not connected")
```

Comment:
What about gathering the operator log at this point in time? Something like this:
```
k8shelper.GetLogsFromNamespace(namespace, testName, utils.TestEnvName())
```